lib/text_parser.dart

```diff
@@ -54,7 +54,7 @@ const SUPPORTED_INLINE_ELEMENTS = <String>{
 };
 */
 
-const SUPPORTED_BLOCK_ELEMENTS = <String>{
+const supportedBlockElements = <String>{
```

Comment:
soru personally doesn't like flutters code style here at all, because with screaming snake case it is immidiately visible that it is a constant global, with just camel case it isn't :/ not sure what to do about that

Reply:
Again, I just followed Flutter's code style...

Reply:
What about overriding it for this case, then?

Reply:
Doesn't seem sound to me, but if you like me to, I can do it.

Reply:
soru thinks she'd prefer that, for both repos